tests/libtest/thrift_client_to_db.py

```diff
@@ -93,16 +93,18 @@ def __exit__(self, type, value, tb):
 
 class CCViewerHelper(ThriftAPIHelper):
 
-    def __init__(self, host, port, product, endpoint,
+    def __init__(self, version, host, port, product, endpoint,
                  auto_handle_connection=True, session_token=None):
         # Import only if necessary; some tests may not add this to PYTHONPATH.
         from libcodechecker import session_manager
-        from codeCheckerDBAccess import codeCheckerDBAccess
-        from codeCheckerDBAccess.constants import MAX_QUERY_SIZE
+
+        if version == '6.0':
```

Comment:
Is this a good idea to hardcode this constant? Is it a good idea to also rely on the minor version here?

Reply:
i agree we should not have this constant here too.

Actually by test cases should be always testing and using only the newest API version.